### PR TITLE
fix scale9sprite global zorder issue.

### PR DESCRIPTION
**Pull request by zilongshanren**
1. it also fix the positiong issue when call setScale9Enabled(false)

### CODE REVIEW

**Comment by xiaofeng11** [comment]
@liamcindy Please review this PR.

**Comment by zilongshanren** [comment]
@dabingnn @super626  please help to review this PR. thanks.

**Comment by liamcindy** [comment]
@zilongshanren @xiaofeng11 It looks good to be merged, thanks.

File: cocos/ui/UIScale9Sprite.cpp [hunk 1]
```diff
@@ -670,7 +670,7 @@ namespace ui {
     
     void Scale9Sprite::draw(Renderer *renderer, const Mat4 &transform, uint32_t flags)
     {
-        if (_scale9Image) {
+        if (_scale9Image && _scale9Enabled) {
```
Comment (dabingnn) [inline cocos/ui/UIScale9Sprite.cpp:673]: if _scale9Enabled is false, it seems nothing will be drawn.

Comment (zilongshanren) [inline cocos/ui/UIScale9Sprite.cpp:673]: @dabingnn  It will be drawn in the visit method.

Comment (dabingnn) [inline cocos/ui/UIScale9Sprite.cpp:673]: Ignore this, I have not seen the visit function.

Comment (zilongshanren) [inline cocos/ui/UIScale9Sprite.cpp:673]: @dabingnn  I change it like this only for back compatibility issue.

**Pull request merged**